app/src/androidTest/java/com/github/se/project/ui/lesson/EditRequestedLessonTest.kt

```diff
@@ -124,30 +126,6 @@ class EditRequestedLessonTest {
     assert(changed)
   }
 
-  @Test
-  fun validateValidatesValidly() {
-    assert(
-        validateLessonInput(
-            "title",
-            "description",
-            mutableStateOf(Subject.AICC),
-            listOf(Language.ENGLISH),
-            "date",
-            "time",
-            1.0,
-            1.0) == null)
-    assert(
-        validateLessonInput(
-            "title",
-            "description",
-            mutableStateOf(Subject.AICC),
-            listOf(Language.ENGLISH),
-            "date",
-            "",
-            1.0,
-            1.0) == "time is missing")
-  }
```

Comment:
Why did you remove this test? Does it fails the CI and so the deletion is temporarily? If it is the case, we should consider having a shared page in the wiki to store them and to not loose it.

Reply:
It is a duplicate: An identical test exists in AddLessonTest